src/pages/Mentorship.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+     <Layout>
+         <b-row>
+      <b-col lg="7" md="7">
+            <h1 class="page-title">{{ $page.main.title }}</h1>
+            <div class="markdown" v-html="$page.main.content" /> 
+         </b-col>
+      <b-col lg="5" md="5">
+
+         
+
+      </b-col>
```

Comment:
Can you put back here as the other pages?

Reply:
I should take out the columns, right?

Reply:
And the row